Scripts/fastlane/Fastfile

```diff
@@ -604,19 +614,28 @@ import "./ScreenshotFastfile"
   # It requires a prebuilt xctestrun file and simulator destination where the tests will be run.
   # -----------------------------------------------------------------------------------
   # Usage:
-  # bundle exec fastlane test_without_building [xctestrun:<Path to xctestrun file>] [destination:<Simulator>] [try_count:<Number of times to try tests>]
+  # bundle exec fastlane test_without_building [name:<Partial name of the .xctestrun file>] [destination:<Simulator>] [try_count:<Number of times to try tests>]
```

Comment:
Wasn't the `destination:` option removed?

Reply:
Oops yep! Thanks!